apps/site/assets/ts/schedule/schedule-loader.tsx

```diff
@@ -56,9 +52,33 @@ const renderSchedulePage = (): void => {
   }
 };
 
+const renderDirection = (schedulePageData: SchedulePageData): void => {
+  const {
+    direction_id: directionId,
+    route_patterns: routePatternsByDirection,
+    shape_map: shapesById,
+    route
+  } = schedulePageData;
+  ReactDOM.render(
+    <ScheduleDirection
+      directionId={directionId}
+      route={route}
+      routePatternsByDirection={routePatternsByDirection}
+      shapesById={shapesById}
+    />,
+    document.getElementById("react-schedule-direction-root")
+  );
+};
+
 const render = (): void => {
   renderMap();
-  renderSchedulePage();
+  const schedulePageDataEl = document.getElementById("js-schedule-page-data");
+  if (!schedulePageDataEl) return;
```

Comment:
fair point. Our front-end error reporting is currently very limited so for the time being this might be safer then potentially have a JS bug on the page that causes unpredictable things to happen. We have a tool called Sentry that we can add to the frontend to collect errors... I think once we add that back we can remove this most-likely unnecessary check